### PR TITLE
Add add/delete function to TagCommand

### PR DESCRIPTION
**Pull request by YeoBohShin**
Users can now add/delete specific tags from students, instead of always replacing tags with a new set of tags

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/DeleteTagCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    }
+
+    @Test
+    public void toStringMethod() {
+        Index targetIndex = Index.fromOneBased(1);
+        DeleteTagCommand deleteTagCommand = new DeleteTagCommand(targetIndex, TypicalPersons.ALICE.getTags());
+        String expected = DeleteTagCommand.class.getCanonicalName() + "{tags=" + TypicalPersons.ALICE.getTags() + "}";
+        assertEquals(expected, deleteTagCommand.toString());
+    }
+}
```
Comment (ChangruHenryQian) [inline src/test/java/seedu/address/logic/commands/DeleteTagCommandTest.java:88]: I like your effort on sufficient testing on deleting tests.

File: src/main/java/seedu/address/logic/commands/DeleteTagCommand.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    private Set<Tag> deleteTags(Set<Tag> studentTags, Set<Tag> tagsToDelete) {
+        Set<Tag> newTags = new HashSet<>(studentTags);
+        newTags.removeAll(tagsToDelete);
+        return newTags;
+    }
+
+    private String generateSuccessMessage(Person personToEdit) {
+        return String.format(MESSAGE_DELETE_TAG_SUCCESS, personToEdit.getName()) + tags;
+    }
+}
```
Comment (ChangruHenryQian) [inline src/main/java/seedu/address/logic/commands/DeleteTagCommand.java:53]: Clear and neat implementation of the deleteTagCommand Class.

**Review by ChangruHenryQian — COMMENTED** [review]
Overall, solid effort for add/delete function to TagCommand. Great job!
Would it be possible for you to make the code pass the Java CI?

**Review by LWZ19 — APPROVED** [review]
LGTM.

**Pull request merged**